src/index.html

```diff
@@ -4,13 +4,55 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      name="search"
+      class="form"
+      data-qa="big"
+    >
+      <input
+        type="search"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="field"
+        data-qa="keypress"
+      >
+      <button type="submit" class="btn search">
+        <img src="images/Search.svg" alt="search-icon">
+      </button>
```

Comment:
It shouldn't be a button, it's just a decoration image, also this button redirects me to page with error